src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -514,6 +516,28 @@ static int M()
         }
 
         [Fact]
+        public void DefaultInEnum()
+        {
+            string source = @"
+enum E
+{
+    DefaultEntry = default,
+    OneEntry = default + 1
```

Comment:
Yes, overload resolution succeeds, then the left and right operands get converted, which gives `default` an `int` type.